students/JaNiece/contact.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="index.css">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title> My Webpage </title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <img src="images/sm.jpeg" class="logo">
+      <a href="index.html"> Home </a>
```

Comment:
🙃 Typically you won't see spaces in between HTML tags and what's inside of them. I.e. you'll usually see `<a href="index.html">Home</a>` instead of `<a href="index.html"> Home </a>`. I think this is because extra spaces like this won't translate to actual spaces in the rendered content in the browser, so it's a little bit clearer to not see the extra spaces in the HTML code.